test/server/lib/DocApi.ts

```diff
@@ -2803,6 +2809,23 @@ function testDocApi(settings: {
           locationSummary: "internal",
         });
       });
+
+      it("POST /docs/{did}/copy fails when the document has external attachments", async function () {
```

Comment:
What about forking?

Reply:
Forking works correctly under the external attachments model - forks use their parent's document pool, so that doesn't need blocking.

I did test with forks and the copy block, and "Save to copy" is also correctly blocked in a document fork.